app/services/LocationService.js

```diff
@@ -63,30 +74,23 @@ export default class LocationServices {
 
         BackgroundGeolocation.configure({
             desiredAccuracy: BackgroundGeolocation.HIGH_ACCURACY,
-            stationaryRadius: 50,
-            distanceFilter: 50,
+            stationaryRadius: 5,
+            distanceFilter: 5,
             notificationTitle: 'Private Kit Enabled',
             notificationText: 'Private Kit is securely storing your GPS coordinates once every five minutes on this device.',
             debug: false,    // when true, it beeps every time a loc is read
-            startOnBoot: false,
+            startOnBoot: true,
             stopOnTerminate: false,
-            locationProvider: BackgroundGeolocation.ACTIVITY_PROVIDER,
-
-            // DEBUG: Use these to get a faster output
-                /*interval: 2000,
-                fastestInterval: 2000, // Time (in milliseconds) between location information polls.  E.g. 60000*5 = 5 minutes
-                activitiesInterval: 2000,*/
+            locationProvider: BackgroundGeolocation.DISTANCE_FILTER_PROVIDER,
 
-            interval: 20000,
-            fastestInterval: 60000 * 5, // Time (in milliseconds) between location information polls.  E.g. 60000*5 = 5 minutes
-            activitiesInterval: 20000,
+            interval: locationInterval,
+            fastestInterval: locationInterval,
+            activitiesInterval: locationInterval,
 
+            activityType: "AutomotiveNavigation",
```

Comment:
We might be knocked on this from for app audit because we are not a navigation app.